commands/pull_request.go

```diff
@@ -208,7 +212,13 @@ func pullRequest(cmd *Command, args *Args) {
 		}
 
 		utils.Check(err)
+
 		pullRequestURL = pr.HTMLURL
+
+		if flagPullRequestAssignee != "" {
+			err = client.UpdateIssue(baseProject, pr.Number, flagPullRequestAssignee)
```

Comment:
would it make sense to include all other fields in this call since it looks like the `client` package offers a more or less 1:1 wrapper around the API ?